src/components/dom/go.js

```diff
@@ -2,10 +2,8 @@ import Link from 'next/link'
 
 const Go = () => {
   return (
-    <Link href='/box' as={`/box`}>
-      <button className='absolute z-20 p-2 m-2 text-white focus:outline-none focus:ring'>
-        Next page
-      </button>
+    <Link href='/box' as={`/box`} >
+        <a className='absolute z-20 p-2 m-2 text-white focus:outline-none focus:ring' >Next page</a>
```

Comment:
replaced the button so it match better in terms of a11y and it doesn't affect the styling